docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -266,88 +309,92 @@ Congratulate and counsel the mother on appropriate IYCF practices.
   * - Death rate
     - Decreases death rate among the treated
     - No
-    - Currently we do not have the relative risk of death of treated/untreated. We will capture some of the effect of tx on death by having the simulants recover faster, therefore leave the high risk state faster. However, by assigning utx simulants the mean death rate when they should have a higher death rate, we might overestimate the protective effect of tx on death.
-  * - Proportion recovered within programme treatment length
+    - Currently we do not have the relative risk of death of treated/untreated. We will capture some of the effect of tx on death by having the simulants recover faster, therefore leave the high risk wasting states faster. However, by assigning tx simulants the mean death rate when they should have a lower death rate, we are killing more than we should in our sim, hence we might be under-estimating the effect of treatment. (For untreated ppl, we are assigning them the mean death rate when in reality they would have a higher death rate. We under-estimate the harm of not-treating)
+  * - Proportion 'effectively covered' within programme treatment length
     - Increases
-    - No (but we can)
-    - We assume that all treatment covered will recover within the time-to-recovery if they had not died. We can modify the treatment coverage by multiplying it by the 'treatment efficacy' giving us the 'effective coverage' instead. 1 - 'effective coverage' will be the 'utx' population. 
-  * - Proportion defaulted during programme
+    - Yes
+    - 'Effective treatment coverage (CE)' is the 'treatment coverage (C)' x 'treatment efficacy (E)'. 1-C is uncovered population. 
+  * - Proportion defaulted/unresponsive during programme
     - Decreases
-    - We can capture this by using 'met-need'
-    - This will decrease as 'met-need' increases.
+    - Yes
+    - (C-CE) The better the programme, the higher the efficacy (E)
```

Comment:
It seems like these two outcomes are really just two ways of saying the same thing: Proportion effectively covered (C) vs. proportion not effectively covered (C-CE).

Would it be better to instead have the affected outcomes be *proportion covered* (C) and *program efficacy* (E)?

Reply:
Actually, I guess the second outcome (C-CE) is equivalent to the efficacy E, but I think it would be clearer if the first outcome were just coverage C rather than effective coverage CE.

Reply:
Yes, you are right. I've revised to make clearer.